src/CommandingOfficers/Commander.java

```diff
@@ -125,12 +126,21 @@ protected void addCommanderAbility(CommanderAbility ca)
   }
 
   /**
-   * Allows a Commander to inject modifications before evaluating a battle.
+   * These functions Allow a Commander to inject modifications before evaluating a battle.
    * Simple damage buffs, etc. can be accomplished via COModifiers, but effects
    * that depend on circumstances that must be evaluated at combat time (e.g. a
    * terrain-based firepower bonus) can be handled here.
+   * applyCombatModifiers() will serve for most combat changes, like the above example.
+   * changeCombatContext() allows the CO to tweak the BattleInstance itself,
+   * to allow for drastic changes to the combat like counterattacking first or at 2+ range.
    */
-  public void applyCombatModifiers(BattleInstance params)
+  public void changeCombatContext(BattleInstance params)
```

Comment:
I wouldn't call the variable `params`, since there's another function with similar usage that takes in a `BattleParams` object.